src/main/resources/avro/common.avdl

```diff
@@ -2,6 +2,325 @@
+with additional sequences to also describe this new observed variation.
+
+Since the added C/G basepair is novel DNA, an additional reference sequence must
+be defined to carry it. Let us call the existing GGTGGNG reference sequence
+"ref" and the new reference sequence "nov1". Thus, the new reference sequence
+"nov1" looks like
+
+```
+    "ref": "nov1":  "ref":
+
+      - 3' 5' -  3' 5' -
+ ...  G       C        C  ...
+ ...  C       G        G  ...
+      - 5' 3' -  5' 3' -
+
+     +1-     +0-      -3+   <- coordinates
+```
+
+In other words, it is a 1-basepair sequence, with a start join to the "-" side
+of "ref"'s basepair at coordinate 1, and an end join to the "-" side of "ref"'s
+basepair at coordinate 3.
+
+The subgraph of our new sequence graph involving "nov1" can be described as a
+single segment:
+
+```
+Segment with top strand C in the "nov1" reference sequence:
+startJoin = (1,-), "ref"
+start = (0,+)
+length = 1
+end = (0,-)
+endJoin = (3, -), "ref"
+
+```
+
+To describe the other new phosphodiester bonds that need to be added to the
+graph, between (2,+) and (4,+) in "ref", we need to create a new reference
+sequence which contains no basepairs at all; it is the empty sequence of DNA
+basepairs. Let the reference sequence "nov2" be such a reference sequence, with
+start and end joins set to describe these bonds. The reference sequence "nov2"
+has no coordinates, and no positions. It looks like this
+
+```
+    "ref": "nov2":  "ref":
+
+      - 3' 5'    3' 5' -
+ ...  A                G  ...
+ ...  T                C  ...
+      - 5' 3'    5' 3' -
+
+     -2+              +4-   <- coordinates
+```
+
+A reference sequence with only start and end joins is describable by a segment
+with only start and end joins. A segment from the "nov2" reference sequence
+would look like this:
+
+```
+Segment with empty top strand in the "nov2" reference sequence:
+startJoin = (2,+), "ref"
+start = null
+length = 0
+end = null
+endJoin = (4,+), "ref"
+```
+All its information will be in its startJoin and endJoin values.
+
+Defined by their top strands, we now have three reference sequences:
+
+```
+ref:    GGTGGNG
+nov1:   C
+nov2:   (empty)
+```
+
+Each has its own 0-based coordinate system, although the coordinate system for
+"nov2" is empty.
+
+The sequence graph formed by these three sequences is described by the following
+set of segments:
+
+```
+1. Segment with top strand GGTGGNG in the "ref" reference sequence:
+startJoin = null
+start = (0,+)
+length = 2
+end = (1,-)
+endJoin = null
+
+2. Segment with top strand C in the "nov1" reference sequence:
+startJoin = (1,-), "ref"
+start = (0,+)
+length = 1
+end = (0,-)
+endJoin = (3,-), "ref"
+
+3. Segment with top strand empty in the "nov2" reference sequence:
+startJoin = (2,+), "ref"
+start = null
+length = 0
+end = null
+endJoin = (4,+), "ref"
+```
+
+It looks like this:
+
+```
+      --C--
+     |     |
+=G==G==T==G==G==N==G=
+      |     |
+       -----
```

Comment:
Also worth noting that illustrations like this don't lay out clearly in ASCII text.